File: Employee Management System(front end)/employee-management/src/app/income-salary.ts
export class IncomeSalary {
    id:number;
    firstName:String;
    lastName:String;
    basicSalary:number;
    attendance:number;
    otherDeductions:number;
    bonus:number


    constructor(
        id: number,
        firstName: string,
        lastName: string,
        basicSalary: number,
        attendance: number,
        otherDeductions: number,
        bonus:number
      ) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.basicSalary = basicSalary;
        this.attendance = attendance;
        this.otherDeductions = otherDeductions;
        this.bonus = bonus;
      }
}

File: Employee-Management-System [Back-end]/java/com/EduBridge/IncomeSalary.java
package com.EduBridge;



import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class IncomeSalary {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String firstName;
    private String lastName;
    private double basicSalary;
    private int attendance;
    private double otherDeductions;

    public IncomeSalary() {
		super();
	}

	

    public IncomeSalary(Long id, String firstName, String lastName, double basicSalary, int attendance,
			double otherDeductions) {
		super();
		this.id = id;
		this.firstName = firstName;
		this.lastName = lastName;
		this.basicSalary = basicSalary;
		this.attendance = attendance;
		this.otherDeductions = otherDeductions;
	}

	
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public double getBasicSalary() {
        return basicSalary;
    }

    public void setBasicSalary(double basicSalary) {
        this.basicSalary = basicSalary;
    }

    public int getAttendance() {
        return attendance;
    }

    public void setAttendance(int attendance) {
        this.attendance = attendance;
    }

    public double getOtherDeductions() {
        return otherDeductions;
    }

    public void setOtherDeductions(double otherDeductions) {
        this.otherDeductions = otherDeductions;
    }
}

